### PR TITLE
Refactor the Network domain to match the proposed architecture - Closes #4208

### LINKED ISSUE #4208: Refactor the Network domain to match the proposed architecture
### Description

As a part of the Make Lisk Desktop blockchain agnostic, we should create a domain for Network. Said domain is responsible for collecting, displaying nodes on the network.

### Acceptance Criteria

According to the TECH proposal document, this domain should have components to:
- List all nodes on the network.
   - No filtering is required.
   - Apply sorting as currently exists.
- Display the geographical distribution of the loaded nodes on a map.
- Display statistical information about the nodes on the network, including:
   - Peer versions distribution pie chart.
   - Peer height distribution pie chart.
   - Peer connectivity distribution pie chart.

### PR DESCRIPTION
**Pull request by vibratious**
### What was the problem?

This PR resolves #4208

### How was it solved?

- [x] Restructured the network directory according to the changes I noticed in #4214, #4240, and #4241.
- [x] Moved network to packages directory
- [x] Updated unit test

### How was it tested?
Tested visually and updated unit tests

### CODE REVIEW

**Review by reyraa — DISMISSED** [review]

File: packages/network/statistics/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { compose } from 'redux';
+import withData from "@common/utilities/withData";
+import { withTranslation } from "react-i18next";
+import Statistics from './statistics';
+import { getNetworkStatistics } from '@network/utilities/api';
+import { tokenMap } from '@token/configuration/tokens';
+
+export default compose(
+  withData({
```
Comment (isalga) [inline packages/network/statistics/index.js:9]: Should Api call not be moved to a separated manager file?

File: packages/network/nodesList/manager/manager.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```
Comment (isalga) [inline packages/network/nodesList/manager/manager.js:1]: That file should be a repeater and not a manager.

File: packages/network/nodesList/manager/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import withData from "@common/utilities/withData";
+import { DEFAULT_LIMIT } from '@views/configuration';
+import { withTranslation } from "react-i18next";
+import withLocalSort from "@common/utilities/withLocalSort";
+import { sortByVersion } from '../../utilities/helpers';
+import NodeList from "./manager";
+import { tokenMap } from '@token/configuration/tokens';
+
+export default compose(
+  withData({
```
Comment (isalga) [inline packages/network/nodesList/manager/index.js:12]: Same, Should not API calls be moved to managers?

**Review by isalga — CHANGES_REQUESTED** [review]

**Comment by ManuGowda** [comment]
@vibratious can you please take a look at the minor review comments and address them? after this we can accept your contribution 🙏🏼 .

File: packages/network/statistics/index.js (hunk 1 above)
Comment (vibratious) [reply packages/network/statistics/index.js:9]: Hi, The API call is already in a dedicated file. And the API utility is already moved to the network domain directory. I'm not sure what you are aiming to change?

File: packages/network/nodesList/manager/manager.js (hunk 2 above)
Comment (vibratious) [reply packages/network/nodesList/manager/manager.js:1]: The Table is performing the task of a repeater. I don't have distinct repeater myself.

File: packages/network/nodesList/manager/index.js (hunk 3 above)
Comment (vibratious) [reply packages/network/nodesList/manager/index.js:12]: Similar to the other file. I think it's fine as it is. but please give more info if you think I have to change it.

File: packages/network/nodesList/manager/manager.js (hunk 2 above)
Comment (soroushm) [reply packages/network/nodesList/manager/manager.js:1]: Dear Elnaz,

we try to separate API call logic from the component so the composer is moved to another file called manager so in future we can change the way of API call or reuse the manager

**Review by soroushm — APPROVED** [review]

**Review by reyraa — APPROVED** [review]

**Pull request merged**